build.yaml

```diff
@@ -14,7 +14,7 @@ cassandra:
   - 2.1
   - 2.2
   - 3.0
-  - 3.10
+  - 3.7
```

Comment:
Sorry I didn't ask further when you mentioned this earlier -- why not test against the latest version?